src/main/kotlin/ch/bfh/habits/SecurityConfiguration.kt

```diff
@@ -0,0 +1,17 @@
+package ch.bfh.habits
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.web.SecurityFilterChain
+
+
+@Configuration
+class SecurityConfiguration {
+    @Bean
+    @Throws(Exception::class)
+    fun filterChain(http: HttpSecurity): SecurityFilterChain? {
+        http.authorizeRequests().anyRequest().permitAll().and().csrf().disable()
```

Comment:
Adapt this later. I would propose to lock down /api to logged in users and put everything under there except login and register.